test/Lokad.ContentAddr.Azure.Tests.csproj

```diff
@@ -16,6 +16,16 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
```

Comment:
The changes to this file should not be included in the PR.

Reply:
I've committed the initial file.